pkg/api/v1/pod/util_test.go

```diff
@@ -525,3 +525,69 @@ func TestUpdatePodCondition(t *testing.T) {
 		assert.Equal(t, test.expected, resultStatus, test.desc)
 	}
 }
+
+func TestHasPodBeenReadyBefore(t *testing.T) {
+	tests := []struct {
+		name                   string
+		isReady                bool
+		readinessChangeRFC3339 string
```

Comment:
why are we making this a string and not a `time.Time`?

Reply:
Didn't want to add time parsing to each test case. Do you think it's better to parse when building test cases?

Reply:
Done what some other tests were doing.